libs/core/date-picker/date-picker.component.html

```diff
@@ -37,7 +37,7 @@
         [disabled]="disabled"
         [isControl]="true"
         [button]="true"
-        [buttonFocusable]="buttonFocusable"
+        [buttonFocusable]="false"
```

Comment:
i think you can set buttonFocusable variable as false and use it instead of set buttonFocusable input directly as false.

Reply:
Yes marking it as deprecated does signal our plan to remove it, but I do like your idea that we should keep the variable here, just make false the default value, so anyone who does still expect the button to be focusable will see that we have deprecated it, but its removal won't be such a surprise